content/src/service/ServiceImpl.ts

```diff
@@ -329,7 +324,24 @@ export class ServiceImpl implements MetaverseContentService {
     // Mark deployment as successful (this does nothing it if hadn't failed on the first place)
     this.components.failedDeploymentsCache.removeFailedDeployment(entity.id)
 
-    return { auditInfoComplete, wasEntityDeployed: !isEntityAlreadyDeployed, affectedPointers }
+    return { auditInfoComplete, wasEntityDeployed: !isEntityAlreadyDeployed }
+  }
+
+  private updateActiveEntities(pointersFromEntity: PointerChanges, entity: Entity) {
+    const { cleared, setted } = Array.from(pointersFromEntity).reduce(
```

Comment:
could we change the name `cleared` and `setted` to  `clearedPointers` and `setPointers` or add its types?